integration_tests/iceberg-cdc/run_test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -ex
+
+# Start test environment.
+docker-compose up -d
```

Comment:
```suggestion
docker-compose up -d --wait
```

Wait for services to be ready.